commands/command_clean.go

```diff
@@ -76,13 +77,13 @@ func clean(gf *lfs.GitFilter, to io.Writer, from io.Reader, fileName string, fil
 		if stat.Size() != cleaned.Size && len(cleaned.Pointer.Extensions) == 0 {
 			Exit("%s\n%s\n%s", tr.Tr.Get("Files don't match:"), mediafile, tmpfile)
 		}
-		Debug("%s exists", mediafile)
+		tracerx.Printf("%s exists", mediafile)
 	} else {
 		if err := os.Rename(tmpfile, mediafile); err != nil {
 			Panic(err, tr.Tr.Get("Unable to move %s to %s", tmpfile, mediafile))
 		}
 
-		Debug(tr.Tr.Get("Writing %s", mediafile))
```

Comment:
We don't want to translate these messages because it is debug output?

Reply:
Exactly.  We don't offer localization of trace log messages (as per [few](https://github.com/git-lfs/git-lfs/blob/d0d1ed06177d78e2222ed3d9a7bb07db6effb033/tq/transfer_queue.go#L384) [random](https://github.com/git-lfs/git-lfs/blob/d0d1ed06177d78e2222ed3d9a7bb07db6effb033/tq/transfer_queue.go#L544) [examples](https://github.com/git-lfs/git-lfs/blob/d0d1ed06177d78e2222ed3d9a7bb07db6effb033/tq/transfer_queue.go#L560) from the transfer queue, just for illustrative purposes.)